cf_xarray/helpers.py

```diff
@@ -338,6 +338,12 @@ def _get_ordered_vertices(
         elif order == "descending":
             endpoints = np.maximum(bounds[..., :, 0], bounds[..., :, 1])
             last_endpoint = np.minimum(bounds[..., -1, 0], bounds[..., -1, 1])
+        elif order == "mixed":
```

Comment:
Even if this is a private function, I would avoid having code that can result in an `UnboundLocalError`.
I suggest either replacing it with a bare else, or adding something like:
```python
else:
    raise NotImplementedError(f"{order = }")
```

Reply:
The new code raises a `ValueError` so it shouldn't result in `UnboundLocalError`. However, I updated this conditional to `else` as a fall-back for any `order` value (even if it should only be ascending, descending, or mixed).